### PR TITLE
Mention coordinated omission in performance docs

### PR DESCRIPTION
**Pull request by maxandersen**
trying to explain to users that when doing performance measurements be aware of coordinated ommision in various tools but noticed we did not mention it in our own performance guide.

here is a suggestion on a way we could inform/educate. feedback welcome!

### CODE REVIEW

**Comment by geoand** [comment]
Thanks, I'll have a close look tomorrow

File: docs/src/main/asciidoc/performance-measure.adoc [hunk 1]
```diff
@@ -9,13 +9,34 @@ include::./attributes.adoc[]
...
 * how we measure memory usage
 * how we measure startup time
 * which additional flags will Quarkus apply to native-image by default
 
 All of our tests are run on the same hardware for a given batch.
 It goes without saying but it's better when you say it.
 
+== Coordinated Omission Problem in Tools
+
+When measuring performance of a framework like Quarkus the latency experience by users are especially interesting and for that there are many different tools. Unfortunately, many fail to measure the latency correctly and instead fall short and create the Coordinate Omission problem. Meaning tools fails to acoomodate for delays to submit new requests when system is under load and aggregate these numbers making the latency and throughput numbers very misleading.
```
Comment (geoand) [inline docs/src/main/asciidoc/performance-measure.adoc:22]:
```suggestion
When measuring performance of a framework like Quarkus, the latency numbers experienced by users are especially interesting and for measuring that, many different tools exist. Unfortunately, many fail to measure the latency correctly and instead suffer from the Coordinate Omission problem - meaning tools fails to accomodate for delays to submit new requests when system is under load and aggregate these numbers, thus rendering their reported latency and throughput numbers misleading.
```

File: docs/src/main/asciidoc/performance-measure.adoc [hunk 2]
```diff
@@ -9,13 +9,34 @@ include::./attributes.adoc[]
...
+
+A good walkthrough of the issue is https://www.youtube.com/watch?v=lJ8ydIuPFeU[this video] where Gil Tene the author of wrk2 explains the issue and https://www.youtube.com/watch?v=xdG8b9iDYbE[Quarkus Insights #22] have John O'Hara from Quarkus performance team show how it can show up.
+
+Although that video and related papers and articles date all back to 2015 then even today you will find tools that fall short with the coordinated oission problem
```
Comment (geoand) [inline docs/src/main/asciidoc/performance-measure.adoc:26]:
```suggestion
Although that video and related papers and articles all date back to 2015, unfortunately many of the widely used tools even today suffer from coordinated oission problem.
```

File: docs/src/main/asciidoc/performance-measure.adoc [hunk 3]
```diff
@@ -9,13 +9,34 @@ include::./attributes.adoc[]
 
 This guide covers:
 
+* Coordinated omission Problem in Tools
```
Comment (Sanne) [inline docs/src/main/asciidoc/performance-measure.adoc:12]: I wouldn't have this as first bullet. In fact I don't think it belongs in this guide - no strong opinion about that, but at least let's not make this aspect too distracting from the primary content.

**Comment by gsmet** [comment]
@maxandersen friendly ping, see especially @Sanne 's comment above.

**Comment by gsmet** [comment]
Closing for lack of activity.

**Pull request closed without merging**